app/views/crime_applications/_review_overview.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <p class="govuk-body-l govuk-!-margin-bottom-2">
-      <%= crime_application.reference %>
+      <span id="reference-number">
+        <%= crime_application.reference %>
+      </span>
+      <%= link_to 'Copy', '', id: 'copy-reference-number', class: 'govuk-link' %>
```

Comment:
class: 'govuk-link--no-visited-state'

Prevents it turning purple after it's clicked

Reply:
@arthurashman thank you I've added this into the code for the copy button 👍